appveyor.yml

```diff
@@ -14,18 +14,34 @@ build_script:
       if ($lastExitCode -ne 0) { throw "Failed to update vcpkg" }
       vcpkg install curl[sspi] expat freetype libjpeg-turbo libogg libpng `
           libtheora libvorbis libvpx openssl opus pcre speex string-theory `
-          zlib --triplet x86-windows-static-dyncrt
+          zlib openal-soft --triplet x86-windows-static-dyncrt

```

Comment:
Looks like these were previously in alphabetical order (makes it easier to scan for/add missing dependencies).

Reply:
:trollface: